kernel/os/pkg.yml

```diff
@@ -29,6 +29,9 @@ pkg.deps:
     - "@apache-mynewt-core/sys/sysinit"
     - "@apache-mynewt-core/util/mem"
 
+pkg.deps.CONSOLE_RTT:
+    - "@apache-mynewt-core/hw/drivers/rtt"
```

Comment:
`RTT` setting is defined by this package so this dependency does not make any sense

Reply:
Indeed error was on my side, and this is not needed